core-services/ipi-deprovision/aws.sh

```diff
@@ -23,6 +23,82 @@ function queue() {
 	"${@}" &
 }
 
+function retry() {
```

Comment:
are we using version 1 or 2 of cli? There is a native retry on CLI, have you tried it?

https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-retries.html

Reply:
https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zi